plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -57,4 +59,29 @@ public void invalidSchemaShouldNotSucceed() throws Exception {
             ve.printStackTrace();
         }
     }
-}
+
+    @Test
+    public void invalidJenkinsDataForBaseConfig() throws Exception {
```

Comment:
@oleg-nenashev Added the tests base configurator wise. Since we have a set number of Root Configurators I decided to write tests around them being invalid and empty. As we go forward we could include missing attributes. But since we have not decided on adding a `required` field to the schema we cannot atm handle missing attributes.

Reply:
How is it related to your change actually? I do not see JSON validation to be invoked in this logic

Reply:
The motivation behind adding this is to check if the user has entered the correct data type for the configurator. If no then the validator will scream at us.